### DISCUSSION TITLE
Two id's are linked with same email account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#79261] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

here was an issue joining the organization: Your GitHub user account @username is currently unlinked. However, you are attempting to authenticate with your Identity Provider using the 'first_lastname@companyname.com' SAML identity which is already linked to a different GitHub user account in the organization. Please reach out to one of your GitHub organization owners for assistance. External ID 'cefc812e-a83f-4ae5-be0b-0862dc84cbf8' is already linked to @username1's account.

**Comment by Mustafa-Hassan2001** [comment]
The issue is that your GitHub user account @username is unlinked, but the SAML identity '[first_lastname@companyname.com](mailto:first_lastname@companyname.com)' is already linked to a different GitHub account in the organization. Contact a GitHub organization owner for assistance.

**Reply by ghost** [reply]
I created support ticket but I didn't hear anything here about support ticket

**Comment by akorp** [comment]
An org owner one can go to `https://github.com/orgs/<your_org>/people/<old_user_account>/sso` to unlink the old account.

**Reply by tttech-gschell** [reply]
thanks that helped me a lot!

**Reply by keithmcgill** [reply]
thanks, this also helped me!

**Reply by sahibbhai** [reply]
Thank youuuu

**Reply by kcarrillorhoads** [reply]
Thank you so much, we had booted the old user so we can't navigate to it through the UI.

**Reply by swapnilguptaTW** [reply]
but in the above link for the user it says that no SSO linked, however the user is still getting the user SSO linked error
<img width="920" alt="Screenshot 2024-09-04 at 1 17 26 PM" src="https://github.com/user-attachments/assets/73521e06-ad23-4fa0-9723-171d11e782c0">

**Reply by ghost** [reply]
Thank you so much! It helped

**Reply by LukeWeldenMobysoft** [reply]
Thanks for this little nugget! 😄

**Reply by mistplay-itadmin** [reply]
@swapnilguptaTW did you find a solution to this? We're having the same issue with one of our users. No SSO identity linked at the Enterprise and Org level, yet he gets the error message that OP posted about.

**Reply by swapnilguptaTW** [reply]
@mistplay-itadmin I do not remember exactly what I did, it was almost a year back. But I think found the solution in other thread. I vaguely recall, in the repository we found the user linked, unlinking the user from there helped.
